sema/subyt/sources.py

```
@@ -87,43 +87,54 @@ def mime_from_identifier(identifier: str) -> str:
         return mimetypes.guess_type(identifier)[0]  # type: ignore
 
     @staticmethod
-    def make_source(identifier: str) -> Source:
-        if validators.url(identifier):
+    def make_source(identifier: str | Path) -> Source:
+        # check for url
+        if check_valid_url(str(identifier)):
             mime: str = SourceFactory.mime_from_remote(identifier)  # type: ignore # noqa
```

Ruff (0.8.2)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)